toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -104,16 +104,21 @@ func resolveGraphNodes(dependencyGraph *pkggraph.PkgGraph, inputSummaryFile, out
 		// Cache an RPM for each unresolved node in the graph.
 		for _, n := range dependencyGraph.AllRunNodes() {
 			if n.State == pkggraph.StateUnresolved {
-				err = resolveSingleNode(cloner, n)
-				if err != nil {
-					errorMessage := strings.Builder{}
-					errorMessage.WriteString(fmt.Sprintf("Failed to resolve all nodes in the graph while resolving '%s'\n", n))
-					errorMessage.WriteString("Nodes which have this as a dependency:\n")
-					for _, dependant := range graph.NodesOf(dependencyGraph.To(n.ID())) {
-						errorMessage.WriteString(fmt.Sprintf("\t'%s' depends on '%s'\n", dependant.(*pkggraph.PkgNode), n))
+				resolveErr := resolveSingleNode(cloner, n)
+				// Failing to clone a dependency should not halt a build.
+				// The build should continue and attempt best effort to build as many packages as possible.
```

Comment:
>The build should continue and attempt best effort to build as many packages as possible. [](start = 7, length = 88)

I actually liked the fact that the build stopped pretty quickly after the first failure - it was easier to find what is the first spec to look for errors. I understand you've got a counter-scenario, so maybe this should be a build variable. What do you think?

Reply:
Is `STOP_ON_PKG_FAIL=y` still an option with the new system? We can look at adding that here later on.

Reply:
`STOP_ON_PKG_FAIL` is still honored in the new tooling. In a future PR I can add that variable to be here as well